lib/facter/letsencrypt_directory.rb

```diff
@@ -0,0 +1,27 @@
+require 'openssl'
+require 'pathname'
+
+Facter.add(:letsencrypt_directory) do
+  setcode do
```

Comment:
since this is not expected to work with windows node, the fact should be confined to Linux kernel.

Reply:
How about BSD kernels? I'd expect it to work there as well.

Reply:
Ah yes ! The module have a huge list of compatibles OSes.
This should work:
`confine :kernel => [:Linux, :FreeBSD, :OpenBSD]`

What i wanted to escape is to run the code of fact on windows.

Reply:
Added the confine. And matched up to whatever rubocop wanted.

Reply:
Whoops I did not squash that right...